### ISSUE TITLE
option to draw multiple graphs on the same axes

### ISSUE THREAD

**Issue by thomasaiman** [martykan/forecastie#249]
I think that condensing the temperature, wind, and rain graphs into a single chart is a great way to display forecast data in the space of a single screen. Having annotated values on the charts may also be helpful. Here's an example of an app that does this: 
https://play.google.com/store/apps/details?id=com.nstudio.weatherhere
![noaagraph](https://user-images.githubusercontent.com/32423251/32151699-f21e2e2e-bcec-11e7-8d7f-457fa50aa02f.png)

Another example from an app that has a UI I really like:
https://sameapk.com/nice-weather/
![niceweather](https://user-images.githubusercontent.com/32423251/32151740-47cb0b26-bced-11e7-85da-46a04a0ce692.png)

Is this something you would be interested in adding?

**Comment by robinpaulson** [comment]
yes. not any time soon though. perhaps you would be prepared to do some coding?

**Comment by thomasaiman** [comment]
I'll put it on my to-do list, but it probably won't happen any time soon haha

**Comment by robinpaulson** [comment]
We're about at a point where we should consider this more seriously. Given the conversation in #397 about adding more graphs, we should include the option to put these on one axis. In the abstract, this is a good move, but it brings the problem of how to indicate values for each line. They can't all use the same y-axis, as the values between them vary wildly. We can reasonably expect at least these ranges:
* Humidity 0 -> 100
* Temperature  -20 -> 40 (C) / -30 -> 100 (F)
* Wind 0 -> 120 (km/h)
* Rain 0 -> 30 (mm)
* UV 0 -> 14
* Pressure 980 -> 1040 (mbar)
And that's only for a subset of the units we support
Not to mention the various pollution charts

* We could decide not to put any information at all on the y-axis, and print the numbers next to to the line, as in the first example above. That will get crowded quickly and trying to automatically shuffle the numbers would be fiddly and irritating.
* Or only print the high and low for the range and leave the user to roughly interpolate the rest.

**Comment by robinpaulson** [comment]
Or perhaps we group them on multiple sets of axes?
* All the "common" information (rain, wind, temperature, pressure) on one graph
* All the pollution + UV on one graph
